src/lib.rs

```diff
@@ -307,6 +307,9 @@ use std::sync::atomic::ATOMIC_USIZE_INIT;
 mod macros;
 mod serde;
 
+#[cfg(feature = "kv_unstable")]
+pub mod key_values;
```

Comment:
Alternative name: mod kv. Do you think this is recognizable and unique and memorable enough?

Reply:
I think I was using the name `kv` at one point, and only switched it to `key_values` so the module name was the same as the `Record::key_values` method. When I first starting exploring `slog`, I found its abbreviated names hard to follow, but those were type names not module names.

I'm on board with `kv`.